### PR TITLE
V0.9.4

### PR DESCRIPTION
**Pull request by Satoshi-Kusumoto**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -754,12 +755,12 @@ construct_runtime! {
...
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
 		Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>},
-		Democracy: pallet_democracy::{Pallet, Call, Storage, Config, Event<T>},
+		Democracy: pallet_democracy::{Pallet, Call, Storage, Config<T>, Event<T>},
 		Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>},
 		TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>},
 		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>},
 		Recovery: pallet_recovery::{Pallet, Call, Storage, Event<T>},
-		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Storage, Inherent, Event<T>, ValidateUnsigned} = 20,
+		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned} = 20,
```
Comment (chenzongxiong) [inline runtime/src/lib.rs:761]: can we remove magic number, like `20`, `21`, `30`

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -11,60 +11,62 @@ serde = { version = "1.0.101", optional = true, features = ["derive"] }
...
-sp-core = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.3" }
-sp-session = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.3" }
-sp-offchain = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.3" }
-sp-block-builder = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.3" }
-sp-transaction-pool = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.3" }
-sp-inherents = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.3" }
-sp-consensus-aura = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.3" }
+sp-std = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.4" }
+sp-api = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.4" }
```
Comment (chenzongxiong) [inline runtime/Cargo.toml:15]: I'm curious about the `default-features` flag. What is the difference between `true` and `false`

File: runtime/src/lib.rs (hunk 1 above)
Comment (Satoshi-Kusumoto) [reply runtime/src/lib.rs:761]: We need to figure out what they are. These are from upstream

File: runtime/Cargo.toml (hunk 2 above)
Comment (Satoshi-Kusumoto) [reply runtime/Cargo.toml:15]: Because `default = ['std']` It means to compile without `std` feature

**Review by suinuj — APPROVED** [review]

**Review by geastwood — COMMENTED** [review]
This is all manual work yeh? We have to keep up with the upstream and manually keep track of these changes?

**Comment by Satoshi-Kusumoto** [comment]
> This is all manual work yeh? We have to keep up with the upstream and manually keep track of these changes?

Yes, basically to do better we even need to keep track of those important upstream PRs

**Pull request merged**